tapir/shifts/views/management.py

```diff
@@ -249,6 +251,10 @@ def form_valid(self, form):
                     required_capabilities=entry.required_capabilities,
                     warnings=entry.warnings,
                 )
+        generate_shifts_up_to(
+            timezone.now().date()
+            + timezone.timedelta(days=len(form.cleaned_data["weekdays"]))
+        )
```

Comment:
This will still generate shifts for all templates, including the ones that we did not touch during duplication. We want to generate shifts only for the templates we just created.